src/pages/MetricDetail.svelte

```diff
@@ -1,9 +1,11 @@
 <script>
   import { getMetricData } from "../state/api";
   import { getMetricBigQueryURL } from "../state/urls";
-
   import Markdown from "../components/Markdown.svelte";
   import NotFound from "../components/NotFound.svelte";
+  import InfoIcon from "../components/icons/InfoIcon.svelte";
+  import HelpHoverable from "../components/HelpHoverable.svelte";
+  import help from "../data/help";
```

Comment:
Let's give this variable a slightly more descriptive name (I know it technically includes links as well, but I think `helpText` is the main outline of what this data is about.

```suggestion
  import helpText from "../data/helpText";
```